docs/contributing.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to browser-compat-data
 
-We're really happy to accept contributions to the mdn-browser-compat-data repository!
+We're really happy to accept contributions to the mdn/browser-compat-data repository!
```

Comment:
Does this need an @?

Reply:
`mdn/browser-compat-data` is still the name of the repo. That part is _not_ changing.

Reply:
We're all good then.